tasks/facial_landmark_detection/search_results.csv

```diff
@@ -0,0 +1,225 @@
+,archid,Partial training Validation Accuracy,onnx_latency (ms),parent,parents,iteration_num,search_walltime_hours,is_pareto,config
+0,8ec1619b,0.04091691207885742,3.053765427062495,,,0,0.7185918717251883,False,"{""arch_def"": [[""ds_r1_k3_s1_c16""], [""ir_r2_k3_s2_e6_c24""], [""ir_r3_k3_s2_e6_c32""], [""ir_r4_k3_s2_e6_c64""], [""ir_r3_k3_s1_e6_c96""], [""ir_r3_k3_s2_e6_c160""], [""ir_r1_k3_s1_e6_c320""]], ""channel_multiplier"": 1.0, ""depth_multiplier"": 1.0}"
```

Comment:
I kinda like how Piero did this where he moved the config json into a separate folder called `archs`, see https://github.com/microsoft/archai/tree/main/tasks/face_segmentation/archs/cpu_target, where the json is also nicely indented.

Reply:
Looks nice indeed. Kind of late to make that change now. Also, this way is meant make it much easier to find correlation between the metrics and the archs, not so convinient with a seperate json file.